### DISCUSSION TITLE
Upset Vector Store: Error: 404 No models loaded.

### DISCUSSION THREAD

**Discussion by ChenZhen1999** [FlowiseAI/Flowise#3157] [Q&A] [answered]
I get this error when I use localAI embeddings for text file upsert: 
Error: vectorsService.upsertVector - Error: Error: 404 No models loaded. Please load a model in the developer page. Please load a model in the developer page. If you don't have any model downloaded, head to app's home page for some options. 
But the parameters of localAI Embeddings are filled out according to the tutorial. I have also confirmed that LMstudio is running fine in the background.
2024-09-06 12:50:58  [INFO] [LM STUDIO SERVER] Supported endpoints:
2024-09-06 12:50:58  [INFO] [LM STUDIO SERVER] ->	GET  http://localhost:1234/v1/models
2024-09-06 12:50:58  [INFO] [LM STUDIO SERVER] ->	POST http://localhost:1234/v1/chat/completions
2024-09-06 12:50:58  [INFO] [LM STUDIO SERVER] ->	POST http://localhost:1234/v1/completions
2024-09-06 12:50:58  [INFO] [LM STUDIO SERVER] ->	POST http://localhost:1234/v1/embeddings

<img width="1913" alt="image" src="https://github.com/user-attachments/assets/97d4d170-29ed-49ee-b17d-da36efc688b2">

But when I use only Chatlocalai it runs fine.
<img width="1911" alt="image" src="https://github.com/user-attachments/assets/012b8218-a2b8-4022-b1d4-c777d6fd0d0a">

**Comment by ctxadm** [comment]
confirm that issue, have the same one!

**Reply by ChenZhen1999** [reply, marked as answer]
Because the selected model does not have Embedding functionality, changing the model will solve the problem.

**Comment by ChenZhen1999** [comment]
Because the selected model does not have Embedding functionality, changing the model will solve the problem.

**Reply by ctxadm** [reply]
thanks a lot for your answer, can you reqommend a Model?
